### PR TITLE
Allow SSIDs containing slashes ...

### LINKED ISSUE #24: sed: bad option in substitution expression
Ich habe einen router mit dem Namen "RouterMitVielZuLangemNamenDerInAllenListenTotalNervt"

unsere SSID ist "http://kiel.freifunk.net/"

Wenn ich auf der Konsole das script aufrufe:

```
/lib/gluon/ssid-changer/ssid-changer.sh
```

kommt dieser Fehler:

```
... node is considered offline
sh: FF_OFFLINE_RouterMi...talNervt: unknown operand
gluon-offline-ssid: TQ is 220, SSID is http://kiel.freifunk.net/, change to FF_OFFLINE_RouterMi...talNervt
sed: bad option in substitution expression
HUP!

```

### PR DESCRIPTION
**Pull request by rubo77**
- fix grammar, indentions and code quality
- prevent unknown operand warning on console

closes https://github.com/ffac/gluon-ssid-changer/issues/24

### CODE REVIEW

File: files/lib/gluon/ssid-changer/ssid-changer.sh [hunk 1]
```diff
@@ -33,14 +33,14 @@ then
 	echo "Gateway TQ is $GATEWAY_TQ node is online"
 	for HOSTAPD in $(ls /var/run/hostapd-phy*); do # check status for all physical devices
 		CURRENT_SSID=`grep "^ssid=$ONLINE_SSID" $HOSTAPD | cut -d"=" -f2`
-		if [ $CURRENT_SSID == $ONLINE_SSID ]
+		if [ "X$CURRENT_SSID" == "X$ONLINE_SSID" ]
```
Comment (mruettgers) [inline files/lib/gluon/ssid-changer/ssid-changer.sh]: `if [ "$CURRENT_SSID" == "$ONLINE_SSID" ]` should be sufficient for casting to string. There's no need to prepend an `X`.

File: files/lib/gluon/ssid-changer/ssid-changer.sh [hunk 2]
```diff
@@ -6,44 +6,44 @@ ONLINE_SSID=$(uci get wireless.client_radio0.ssid -q)
...
 fi
 
 if [ $GATEWAY_TQ -gt $UPPER_LIMIT ];
 then
 	echo "Gateway TQ is $GATEWAY_TQ node is online"
-	for HOSTAPD in $(ls /var/run/hostapd-phy*); do #Check status for all physical devices
+	for HOSTAPD in $(ls /var/run/hostapd-phy*); do # check status for all physical devices
 		CURRENT_SSID=`grep "^ssid=$ONLINE_SSID" $HOSTAPD | cut -d"=" -f2`
-		if [ $CURRENT_SSID == $ONLINE_SSID ]
+		if [ "X$CURRENT_SSID" == "X$ONLINE_SSID" ]
```
Comment (mruettgers) [inline files/lib/gluon/ssid-changer/ssid-changer.sh]: `if [ "$CURRENT_SSID" == "$ONLINE_SSID" ]` should be sufficient for casting to string. There's no need to prepend an `X`.

File: files/lib/gluon/ssid-changer/ssid-changer.sh [hunk 3]
```diff
@@ -6,44 +6,44 @@ ONLINE_SSID=$(uci get wireless.client_radio0.ssid -q)
...
 		then
 			echo "SSID $CURRENT_SSID is correct, noting to do"
 			HUP_NEEDED=0
 			break
 		fi
 		CURRENT_SSID=`grep "^ssid=$OFFLINE_SSID" $HOSTAPD | cut -d"=" -f2`
-		if [ $CURRENT_SSID == $OFFLINE_SSID ]
+		if [ "X$CURRENT_SSID" == "X$OFFLINE_SSID" ]
```
Comment (mruettgers) [inline files/lib/gluon/ssid-changer/ssid-changer.sh]: See above

File: files/lib/gluon/ssid-changer/ssid-changer.sh [hunk 4]
```diff
@@ -54,34 +54,34 @@ fi
 if [ $GATEWAY_TQ -lt $LOWER_LIMIT ];
 then
 	echo "Gateway TQ is $GATEWAY_TQ node is considered offline"
-	for HOSTAPD in $(ls /var/run/hostapd-phy*); do #Check status for all physical devices
-		CURRENT_SSID=`grep "^ssid=$OFFLINE_SSID" $HOSTAPD | cut -d"=" -f2`
-		if [ $CURRENT_SSID == $OFFLINE_SSID ]
+	for HOSTAPD in $(ls /var/run/hostapd-phy*); do # check status for all physical devices
+		CURRENT_SSID="$(grep "^ssid=$OFFLINE_SSID" $HOSTAPD | cut -d"=" -f2)"
+		if [ "X$CURRENT_SSID" == "X$OFFLINE_SSID" ]
```
Comment (mruettgers) [inline files/lib/gluon/ssid-changer/ssid-changer.sh]: See above

**Comment by rubo77** [comment]
Ich habe die commits mal ordentlich sortiert. kann dies gemerged werden?

**Comment by rubo77** [comment]
rebased, so there are no conflicts

**Pull request merged**